packages/ui/src/lib/sidebar/DefaultHood.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang='ts'>
+	import Menu from '@rgossiaux/svelte-heroicons/solid/Menu';
+	import { isSidebarOpen } from './sidebarState'
```

Comment:
```suggestion
	import { isSidebarOpen } from './sidebarState'
```

Reply:
This dependency is needed. I expect the implementation of this component will change in the near future though.